SparkPost/2web2ui
# SA-770 - Display results by seed addresses for mailbox provider

### What Changed
 - Update StopTest component to be more independent. Moved the dispatched calls into the component rather than being passed as a prop.
 - Updated the InfoBox component to have custom css styling. Also removed the break tag as it didn't seem to affect the current implementations. 
 - Added new routes and action/reducers for the all messages page /endpoint. 
 - Created the all messages page. Currently only supports mailbox-providers but code for future sending ip support is included. 
 - Added link to the all messages page in the details page in the breakdown by mailbox provider section

### How To Test
 - Go to /inbox-placement. Click on any (real) test. On the details page, scroll down and click on one of the mailbox-providers. Verify that the new page looks like the mocks. 

### To Do
- [ ] UX feedback
